### PR TITLE
Improve the installation instructions

### LINKED ISSUE #144: Add a note about updating conda in episode installation
updating conda speeds up the installation.

### PR DESCRIPTION
**Pull request by SarahAlidoost**
# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [x] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [ ] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [x] Give this pull request a descriptive title.
- [ ] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [x] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [x] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [x] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes #144

### CODE REVIEW

File: _episodes/02-installation.md [hunk 1]
```diff
@@ -22,61 +22,57 @@ be found in the
 We will first install Conda, and then ESMValTool. We end this chapter by testing
 that the installation was successful.
 
-## Install Conda
+## Install ESMValTool on Windows
```
Comment (Peter9192) [inline _episodes/02-installation.md:25]: Maybe we could make a call-out box out of this?

**Review by Peter9192 — CHANGES_REQUESTED** [review]
Looks good! 1 small suggestions.

File: _episodes/02-installation.md [hunk 2]
```diff
@@ -22,61 +22,57 @@ be found in the
...
+```
 
-4.  You will need to restart your terminal for the changes to have effect.
+If conda is installed, we will see a list of packages.
+We recommend updating conda before esmvaltool installation. To do so, run:
 
-5.  Verify you have a working conda installation by listing all installed
-    packages
+```bash
+conda update
```
Comment (Peter9192) [inline _episodes/02-installation.md:53]: Should this be `conda update -n base conda`?

Comment (SarahAlidoost) [reply _episodes/02-installation.md]: @Peter9192 thanks. It is fixed.

File: _episodes/02-installation.md (hunk 1 above)
Comment (SarahAlidoost) [reply _episodes/02-installation.md]: good idea, done.

File: _episodes/02-installation.md [hunk 3]
```diff
@@ -9,31 +9,35 @@ objectives:
...
-
-After installing the WSL, installation can be done using the same instructions for ``Linux``.
-
+> ## Install ESMValTool on Windows
+>
+> ESMValTool does not directly support Windows, but successful usage has been
+> reported through the [Windows Subsystem for
+> Linux(WSL)](https://docs.microsoft.com/en-us/windows/wsl/),
+> available in Windows 10.
+>
```
Comment (Peter9192) [inline _episodes/02-installation.md:34]:
```suggestion
```

File: _episodes/02-installation.md [hunk 4]
```diff
@@ -9,31 +9,35 @@ objectives:
...
+> To install the WSL please follow the instructions [on the Windows Documentation
+> page](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+>
```
Comment (Peter9192) [inline _episodes/02-installation.md:37]:
```suggestion
```

Comment (Peter9192) [reply _episodes/02-installation.md:37]: I'd suggest to remove these blank lines, that makes the callout box look a bit weird.

**Review by Peter9192 — APPROVED** [review]

**Pull request merged**